### BIG differences. This is diff between CLUS "System Class STRING"
### and CLHS selection topic
### http://www.ai.mit.edu/projects/iiip/doc/CommonLISP/HyperSpec/Body/any_string.html
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:types:string.txt	2017-04-27 21:41:22.000000000 +0100
+++ saved-pages-clus/cl:types:string.txt	2017-04-27 21:41:22.000000000 +0100
@@ -1,4 +1,52 @@
-STRING
-Please select which reference to STRING you intended
+System Class STRING
 
-FunctionSystem ClassGlossary Entry
\ No newline at end of file
+Class Precedence List
+string, vector, array, sequence, t
+Description
+A string is a specialized vector whose elements are of type character or a subtype of type character. When used as a type specifier for object creation, string means (vector character).
+Compound Type Specifier Kind
+Abbreviating.
+Compound Type Specifier Syntax
+string [size]
+Compound Type Specifier Arguments
+ size - a non-negative fixnum, or the symbol *.
+Compound Type Specifier Description
+This denotes the union of all types (array c (size)) for all subtypes c of character; that is, the set of strings of size size.
+See Also
+ {\secref\StringConcepts}
+ {\secref\Doublequote}
+ {\secref\PrintingStrings}
+\
\ No newline at end of file
